Makefile

```diff
@@ -9,9 +9,15 @@ $(VIRTUALENV)/.installed:
 	virtualenv --python $(PYTHON_VERSION) $(VIRTUALENV)
 	$(VIRTUALENV)/bin/pip3 install -r requirements.txt
 	$(VIRTUALENV)/bin/pip3 install -r requirements_test.txt
+	$(VIRTUALENV)/bin/pip3 install -r docs/requirements.txt # Installs requirements to docs
 	$(VIRTUALENV)/bin/pip3 install -e .
 	touch $@
 
+.PHONY: update-docs
+update-docs:
+	cd docs && ../$(VIRTUALENV)/bin/sphinx-apidoc --no-toc -d 5 -H WellcomeML -o . -f ../wellcomeml
```

Comment:
can this not run without cd ing?

Reply:
You are right, we can. I changed that in the new commit. The next command though depends on a makefile in the sub-directory so to run without cd it would require bigger changes, so I left it like that.